<launch>
  <arg name="debug" default="false"/>
  <!--<arg name="debug_valgrind" default="false"/>-->

  <arg name="prefix" value="" unless="$(arg debug)"/>
  <arg name="prefix" value="gdb -ex run --args" if="$(arg debug)"/>

  <node pkg="mpl_test_node"
    type="rss_evaluate4"
    name="test_primitive"
    launch-prefix="$(arg prefix)"
    output="screen">
    <remap from="~replan" to="/replan"/>
    <!-- Set start and goal -->
    <param name="start_x" value="1.0"/>
    <param name="start_y" value="5.0"/>
    <param name="start_z" value="0.1"/>
    <param name="start_vx" value="0.0"/>
    <param name="start_vy" value="0.0"/>
    <param name="start_vz" value="0.0"/>
    <param name="goal_x" value="9.0"/>
    <param name="goal_y" value="5.0"/>
    <param name="goal_z" value="0.06"/>
    <!-- Set dynamic constraints -->
    <param name="v_max" value="2.0"/>
    <param name="a_max" value="1.0"/>
    <param name="j_max" value="1.0"/>
    <param name="dt" value="0.5"/>
    <param name="ndt" value="100"/>
  </node>

</launch>
